internal/gcs/protocol.go

```diff
@@ -115,6 +115,19 @@ func (typ msgType) String() string {
 type requestBase struct {
 	ContainerID string    `json:"ContainerId"`
 	ActivityID  guid.GUID `json:"ActivityId"`
+
+	// TraceID is the OpenCensus TraceID to propagate to the guest.
+	//
+	// NOTE: This is not a part of the protocol but because its a JSON protocol
+	// adding fields is a non-breaking change. If the guest supports it this is
+	// just additive context.
+	TraceID string `json:",omitempty"`
```

Comment:
@kevpar  - This will only work for external bridge but this will forward the context to the OpenGCS. Did I miss anything?

Reply:
I wonder if you should name this OpenCensusTraceID, etc. since the format for OpenTelemetry may be different in the future.

Reply:
That's a good idea. I will update.